packages/victory-line/src/v37/victory-line.tsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Curve, CurveProps } from "../curve";
+import {
+  InterpolationPropType,
+  useData,
+  useDomain,
+  useScale,
+  VictoryClipContainer,
+  VictoryClipContainerProps,
+  VictoryCommonProps,
+  VictoryContainer,
+  VictoryDatableProps,
+  VictoryLabel,
+  VictoryLabelableProps,
+  VictoryLabelProps,
+  VictoryStyleInterface,
+  VictoryTheme,
+  withContainer,
+} from "victory-core";
+import { Clone } from "./clone";
+
+export interface VictoryLineProps
+  extends VictoryCommonProps,
+    VictoryDatableProps,
+    VictoryLabelableProps {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  interpolation?: InterpolationPropType | Function;
+  samples?: number;
+  style?: VictoryStyleInterface;
+  animate?: boolean;
+}
+
+const propTypes: PropTypes.ValidationMap<VictoryLineProps> = {
+  interpolation: PropTypes.oneOfType([
+    PropTypes.oneOf([
+      "basis",
+      "bundle",
+      "cardinal",
+      "catmullRom",
+      "linear",
+      "monotoneX",
+      "monotoneY",
+      "natural",
+      "step",
+      "stepAfter",
+      "stepBefore",
+    ] as const),
+    PropTypes.func,
+  ]).isRequired,
+};
+
+const defaultProps: Required<
+  Pick<
+    VictoryLineProps,
+    | "containerComponent"
+    | "data"
+    | "dataComponent"
+    | "interpolation"
+    | "groupComponent"
+    | "labelComponent"
+    | "samples"
+    | "sortKey"
+    | "sortOrder"
+    | "standalone"
+    | "theme"
+  >
+> = {
+  data: [
```

Comment:
I haven't tried running this yet, but does it work to pass default data in here? Or is the default data coming from the provider defaults?

Reply:
Good question ...
In the next PR #2356 , I've done some work to make sure the `defaultProps` is assigned to the top-most wrapper.  This ensures that all "wrappers" (including the VictoryProvider, if needed) will be able to use these default props.  
And if there's a "parent provider" in the DOM, these default props will be properly sync'd with the parent.